bin/kafka-avro-console-consumer

```diff
@@ -16,7 +16,13 @@
 
 base_dir=$(dirname $0)/..
 
-CLASSPATH=$CLASSPATH:$base_dir/share/java/avro-serializer/*
+# Production jars
+export CLASSPATH=$CLASSPATH:$base_dir/share/java/kafka-serde-tools/*
+
+# Development jars. `mvn package` should collect all the required dependency jars here
```

Comment:
As @ewencp, we are currently suffering from the fact that schema-registry (server/core + client) is currently in the same repository as the serde tools (e.g. Avro serializer).  We already talked about reconsidering this setup once 2.0.0 is released.